packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIcon.tsx

```diff
@@ -8,44 +8,108 @@ import { useTheme } from '~components/BladeProvider';
 import Svg, { Path } from '~components/Icons/_Svg';
 import { getIn } from '~utils';
 
-const CheckedIcon = ({ color }: { color: string }) => {
+const CheckedIconSmall = ({ color }: { color: string }) => {
   return (
-    <Svg width="16px" height="16px" viewBox="0 0 16 16" fill="none">
+    <Svg width="8" height="8" viewBox="0 0 8 8" fill="none">
       <Path
         fillRule="evenodd"
         clipRule="evenodd"
-        d="M12.3536 4.64645C12.5488 4.84171 12.5488 5.15829 12.3536 5.35355L6.85355 10.8536C6.65829 11.0488 6.34171 11.0488 6.14645 10.8536L3.64645 8.35355C3.45118 8.15829 3.45118 7.84171 3.64645 7.64645C3.84171 7.45118 4.15829 7.45118 4.35355 7.64645L6.5 9.79289L11.6464 4.64645C11.8417 4.45118 12.1583 4.45118 12.3536 4.64645Z"
+        d="M6.90237 1.76413C7.03254 1.89431 7.03254 2.10536 6.90237 2.23554L3.2357 5.90221C3.10553 6.03238 2.89447 6.03238 2.7643 5.90221L1.09763 4.23554C0.967456 4.10536 0.967456 3.89431 1.09763 3.76414C1.22781 3.63396 1.43886 3.63396 1.56904 3.76414L3 5.1951L6.43096 1.76413C6.56114 1.63396 6.77219 1.63396 6.90237 1.76413Z"
         fill={color}
+        stroke="white"
+        strokeWidth="0.5"
+        strokeLinecap="round"
+        strokeLinejoin="round"
       />
     </Svg>
   );
 };
 
-const IndeterminateIcon = ({ color }: { color: string }) => {
+const CheckedIconMedium = ({ color }: { color: string }) => {
   return (
-    <Svg width="16px" height="16px" viewBox="0 0 16 16" fill="none">
+    <Svg width="12" height="12" viewBox="0 0 12 12" fill="none">
```

Comment:
We're removing `px` now in height / width?

Reply:
Good catch! I'll add it.